lib.go

```diff
@@ -212,6 +228,12 @@ func (vm *VM) Query(
 		return nil, gasUsed, err
 	}
 
+	gasForDeserialization := uint64(deserCost.Mul(uint64(len(data))).Int())
```

Comment:
This is not needed anymore, right?

```suggestion
	gasForDeserialization := deserCost.Mul(uint64(len(data))).Int()
```